guide/getting-started.md

```diff
@@ -13,6 +13,21 @@ To use OpenIddict, you need to:
     <PackageReference Include="OpenIddict.EntityFrameworkCore" Version="2.0.0-*" />
     ```
 
+  - **OPTIONAL: If you want to try out the latest features and bug fixes,** there is a MyGet feed with nightly builds
+    of OpenIddict.
+
+    To reference the OpenIddict MyGet feed, **create a `NuGet.config` file** (at the root of your solution):
+
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+    <packageSources>
```

Comment:
nit: can you please a level of indentation to be consistent with the snippet you added in the main repo?

Reply:
oops, I didn't notice VS Code changed the indentation when I pasted it :D will fix